README.md

```diff
@@ -1,159 +1,97 @@
+For more information about the library, check out the [Fast-RTPS documentation](https://fast-rtps.docs.eprosima.com/en/latest/).
+You can find all the library's source code on our [GitHub repository](https://github.com/eProsima/Fast-RTPS).
 
 ## Installation Guide
-You can get either a binary distribution of *eprosima Fast RTPS* or compile the library yourself from source.
 
-### Installation from binaries
-The latest, up to date binary release of *eprosima Fast RTPS* can be obtained from the <a href='http://www.eprosima.com'>company website</a>.
+1. In order to build and test the documentation, some dependencies must be installed beforehand:
 
-### Installation from Source
-
-#### Dependencies
-
-##### Asio and TinyXML2 libraries
-
-On Linux, you can install these libraries using the package manager of your Linux distribution.
-For example, on Ubuntu you can install them by using its package manager with the next command.
-
-```bash
-sudo apt install libasio-dev libtinyxml2-dev
-```
+    ```bash
+    sudo apt update
+    sudo apt install -y \
+        git \
+        gcc \
+        g++ \
+        cmake \
+        curl \
+        wget \
+        libasio-dev \
+        libtinyxml2-dev \
+        doxygen \
+        python3 \
+        python3-pip \
+        python3-venv \
+        python-sphinxcontrib.spelling=4.0.1-1 \
+        imagemagick=8:6.9.7.4+dfsg-16ubuntu6.8
+    ```
 
-On Windows, you can install these libraries using [Chocolatey](https://chocolatey.org).
-First, download the following chocolatey packages from this
-[ROS2 Github repository](https://github.com/ros2/choco-packages/releases/latest).
+1. Clone the repository
 
-* asio.1.12.1.nupkg
-* tinyxml2.6.0.0.nupkg
+    ```bash
+    cd ~
+    git clone https://github.com/eProsima/Fast-RTPS-docs fastrtps-docs
+    ```
 
-Once these packages are downloaded, open an administrative shell and execute the following command:
+1. Create a virtual environment and install python3 dependencies
 
-```batch
-choco install -y -s <PATH\TO\DOWNLOADS\> asio tinyxml2
-```
+    ```bash
+    cd ~/fastrtps-docs
+    python3 -m venv fastrtps-docs-venv
+    source fastrtps-docs-venv/bin/activate
+    pip3 install -r docs/requirements.txt
+    ```
 
-Please replace `<PATH\TO\DOWNLOADS>` with the folder you downloaded the packages to.
+## Getting Started
 
-#### Colcon installation
-*******************
-[colcon](https://colcon.readthedocs.io) is a command line tool to build sets of software packages.
-This section explains to use it to compile easily Fast-RTPS and its dependencies.
-First install ROS2 development tools (colcon and vcstool):
+To generate the documentation in a HTML format for a specific branch of Fast-RTPS run:
 
 ```bash
-pip install -U colcon-common-extensions vcstool
+cd ~/fastrtps-docs
+source fastrtps-docs-venv/bin/activate
+FASTRTPS_BRANCH=<branch> make html
```

Comment:
This looks like the branch variable is mandatory.

I would start with a command without specifying the Fast-RTPS branch (i.e., using master, that will be most useful for users) and then explain how to use another branch different form master

Reply:
done